.github/workflows/build.yml

```diff
@@ -0,0 +1,122 @@
+      BUILD_CONFIGURATION: 'Release'
+
+    steps:
+
+    - uses: actions/checkout@v1
+      name: Checkout
+
+    - name: Setup dotnet
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: | 
+          6.0.x
+
+    - name: Setup Node.js (${{ matrix.node_version }})
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node_version }}
+
+    - name: Variables
+      run: |
+        echo VERSION=$(npm run version --silent) >> $GITHUB_ENV
+      shell: bash
+
+    # - name: Variables
+    #   run: |
+    #     echo "::set-env name=VERSION::$(npm run version --silent)"
+    #     echo "::set-env name=RELEASE_CONFIGURATION::${{ matrix.runtime }}"
+
+    # - name: Setup .NET Core (${{ matrix.dotnet_version }})
+    #   uses: actions/setup-dotnet@v1
+    #   with:
+    #     dotnet-version: ${{ matrix.dotnet_version }}
+
+    - name: Restore
+      run: dotnet restore ${{env.SOLUTION_PATH}}
+
+    - name: Unit Test
+      run: dotnet test -v=normal -c ${{env.BUILD_CONFIGURATION}}
+
+    # - name: Build
+    #   run: dotnet build -c ${{env.BUILD_CONFIGURATION}} -r ${{env.RELEASE_CONFIGURATION}} /p:Version=${{ env.VERSION }} -v m ${{env.SOLUTION_PATH}}
+
+    - name: Publish (Blockcore Dns)
+      run: dotnet publish -c ${{env.BUILD_CONFIGURATION}} -r ${{matrix.runtime}} /p:Version=${{ env.VERSION }} -v m -o ./src/${{ env.PROJECT_NAME }}/bin/publish ${{env.PROJECT_PATH}}
+
+    - name: Package Name
+      run: |
+        echo RELEASE_NAME=${{ env.PROJECT_NAME }}-${{ env.VERSION }}-${{ matrix.runtime }}${{ matrix.extension }} >> $GITHUB_ENV
+        echo RELEASE_PATH=${{ env.PROJECT_NAME }}-${{ env.VERSION }}-${{ matrix.runtime }}${{ matrix.extension }} >> $GITHUB_ENV
+      shell: bash
+
+    - name: Package (Linux)
+      if: matrix.os == 'ubuntu-latest'
+      run: |
+        echo RELEASE_PATH=./src/${{env.PROJECT_NAME}}/bin/publish/${{env.RELEASE_NAME}} >> $GITHUB_ENV
+        cd ./src/${{env.PROJECT_NAME}}/bin/publish/
+        tar cvzf ${{env.RELEASE_NAME}} *
+        cd ../../../../
+
+    - name: Package (Windows)
+      if: matrix.os == 'windows-latest'
+      run: |
+        Compress-Archive -Path .\src\${{env.PROJECT_NAME}}\bin\publish\* -DestinationPath .\${{env.RELEASE_NAME}}
+
+    - name: Package (Mac)
+      if: matrix.os == 'macos-latest'
+      run: |
+        zip -r ${{env.RELEASE_NAME}} ./src/${{env.PROJECT_NAME}}/bin/publish/
+
+    #- run: mkdir -p path/to/artifact
+    #- run: echo hello > path/to/artifact/world.txt
+
+    - uses: actions/upload-artifact@v1
+      with:
+        name: blockcore-Dns
+        path: "${{env.RELEASE_PATH}}"
+
+    - name: Release
+      uses: sondreb/action-release@main
+      with:
+        commit: 'master'
+        token: ${{ secrets.GITHUB_TOKEN }}
+        files: "${{env.RELEASE_PATH}}"
+        draft: true
+        prerelease: false
+        body: ''
+        name: "Blockcore Dns (Release ${{env.VERSION}})"
```

Comment:
Is name "Blockcore DNS" or "Blockcore Dns"?

Reply:
I suppose for consistency `Dns`